schema/mysql-migrations/1.0.0.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE template SET ctime = UNIX_TIMESTAMP() * 1000.0, mtime = UNIX_TIMESTAMP() * 1000.0;
+
+UPDATE report SET ctime = UNIX_TIMESTAMP() * 1000.0, mtime = UNIX_TIMESTAMP() * 1000.0;
+
+UPDATE reportlet SET ctime = UNIX_TIMESTAMP() * 1000.0, mtime = UNIX_TIMESTAMP() * 1000.0;
+
+UPDATE config SET ctime = UNIX_TIMESTAMP() * 1000.0, mtime = UNIX_TIMESTAMP() * 1000.0;
+
+UPDATE schedule SET start = UNIX_TIMESTAMP() * 1000.0;
+UPDATE schedule SET ctime = UNIX_TIMESTAMP() * 1000.0, mtime = UNIX_TIMESTAMP() * 1000.0;
```

Comment:
That way you're setting all columns to the current `UNIX_TIMESTAMP()`, and overwriting the actual values. As the columns are already stored in seconds, you don't need to do anything else but multiply the current values by `1000.0`.

```sql
UPDATE template SET ctime = ctime * 1000.0, mtime = mtime * 1000.0;
```